TarSCM/scm/git.py

```diff
@@ -251,7 +251,7 @@ def prepare_working_copy(self):
         # between multiple services
         org_clone_dir  = self.clone_dir
         self.clone_dir = self.repodir
-        command = ['git', 'clone', self.repocachedir, self.url, self.clone_dir]
+        command = ['git', 'clone', '--no-checkout']
```

Comment:
No, AFAICS it works as expected :-). I think the "osc localrun" might be not necessary but anyway this is how I would have expected. We should check if there is already a test case for this and document it here.